src/modules/app/containers/RouterFragment.js

```diff
@@ -13,54 +13,80 @@ import PdfFetcherPage from 'routes/pdf-fetcher/containers/PdfFetcherPage'
 import MainDisclaimerPage from 'routes/main-disclaimer/components/MainDisclaimerPage'
 import LandingPage from 'routes/landing/containers/LandingPage'
 import CategoriesPage from 'routes/categories/containers/CategoriesPage'
+import I18nRedirect from 'modules/app/containers/I18nRedirect'
+import PropTypes from 'prop-types'
+import RouteConfig from '../RouteConfig'
 
 /**
  * todo: Test and document in WEBAPP-90
  * todo: Layouts should be set in each route
  */
 class RouterFragment extends React.Component {
+  static propTypes = {
+    routerConfig: PropTypes.instanceOf(RouteConfig).isRequired
+  }
+
+  static isLanguageCode (language) {
+    return language && language.length === 2
+  }
+
   render () {
+    const {routerConfig} = this.props
+    const matchRoute = (id) => routerConfig.matchRoute(id)
```

Comment:
Just a hint for pureComponents: Lambda's defined inside the render() method also trigger a re-render of all pure-components receiving them as prop. (Doesn't matter in this case, since RouterFragment shouldn't re-render a lot itself, since its property doesn't change.)

Reply:
Done